File: Signiture-Backend/delivery/api/src/main/java/pcy/study/api/domain/user/model/UserDetails.java
package pcy.study.api.domain.user.model;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;

@Builder
public record UserDetails(
        @Schema(description = "식별값", example = "1L")
        Long id,
        @Schema(description = "이름", example = "이름")
        String name,
        @Schema(description = "이메일", example = "example@gmail.com")
        String email,
        @Schema(description = "주소", example = "서울특별시 강남구 xxx")
        String address
) {
}

File: Signiture-Backend/kotlin/delivery/api/src/main/kotlin/pcy/study/api/domain/user/model/UserDetails.kt
package pcy.study.api.domain.user.model

import io.swagger.v3.oas.annotations.media.Schema

data class UserDetails(
    @field:Schema(description = "식별값", example = "1")
    val id:Long,
    @field:Schema(description = "이름", example = "이름")
    val name:String,
    @field:Schema(description = "이메일", example = "example@gmail.com")
    val email:String,
    @field:Schema(description = "주소", example = "서울특별시 강남구 xxx")
    val address:String,
)